src/aleph/vm/orchestrator/views/__init__.py

```diff
@@ -169,6 +177,25 @@ async def status_check_fastapi(request: web.Request):
         return web.json_response(result, status=200 if all(result.values()) else 503)
 
 
+async def status_check_host(request: web.Request):
+    """Check that the platform is supported and configured correctly"""
+
+    result = {
+        "ipv4": {
+            "egress": await check_host_egress_ipv4(),
+            "dns": await check_dns_ipv4(),
+            "domain": await check_domain_resolution_ipv4(),
+        },
+        "ipv6": {
+            "egress": await check_host_egress_ipv6(),
+            "dns": await check_dns_ipv6(),
+            "domain": await check_domain_resolution_ipv6(),
+        },
+    }
```

Comment:
Maybe we need to improve the performance of that method just calling the method `resolve_dns` just twice instead of calling it for every field.

Reply:
It is only called 2 times. It is indeed not ideal in terms of performance, but I didn't find a way to make the code simpler without this duplication. Any idea ?

Reply:
It can be to call the method `resolve_dns` on `status_check_host` and deal with the result on the other methods.